.github/workflows/build-pr.yaml

```diff
@@ -2,34 +2,30 @@ name: Build PRs
 on:
   pull_request:
 jobs:
-  build-all:
+  run-scripted-tests:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
I don't think we want to avoid the `fail-fast` semantic here

Reply:
Fixed here: 637ca491f5b3efe7457d9186e6187d8ae9179e4d